recipes/autogen/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "5.18.7" %}
+
+package:
+  name: autogen
+  version: {{ version }}
+
+source:
+  fn: autogen-5.18.7.tar.xz
+  url: http://ftpmirror.gnu.org/autogen/autogen-5.18.7.tar.gz
+  sha1: 83dc95438470447b13f9b958871d7c3b7a868733
+
+build:
+  number: 0
+  skip: True # [win]
+
+requirements:
+  build:
+    - guile # [not win]
+    - pkg-config # [not win]
+
+test:
+  commands:
+    - autogen --help
+
+about:
+  home: http://autogen.sourceforge.net
+  license: BSD
+  summary: AutoGen is a tool designed to simplify the creation and maintenance of programs that contain large amounts of repetitious text.
+
+extra:
+  recipe-maintainers:
+    - stefan-balke
```

Comment:
Here is where you can get in tough with others interested into help you maintain this recipe. Every GitHub handle listed here will be automatically added as part of the team. (So it is nice to ask them first to avoid spamming them with automatic GitHub emails :wink:)